package.json

```diff
@@ -22,13 +22,15 @@
     "@heroicons/react": "1.0.2",
     "@netlify/plugin-nextjs": "3.7.1",
     "@tailwindcss/typography": "0.4.1",
+    "@types/nprogress": "^0.2.0",
```

Comment:
`@types` should be as a `devDependencies`

Reply:
Updated